clients/unshaded/src/main/java/tachyon/client/file/FileOutStream.java

```diff
@@ -258,10 +258,8 @@ private void getNextBlock() throws IOException {
       try {
         WorkerNetAddress address = mLocationPolicy.getWorkerForNextBlock(
```

Comment:
This is guaranteed to return a non-null address? (The previous code checked for null)